### PR TITLE
BZ1891361: Added note block about vSphere volume is not deleted after  the cluster is destroyed

### PR DESCRIPTION
**Pull request by darshan-nagaraj**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1891361

OCP version: 4.6+

Direct Doc Preview Link: https://deploy-preview-42447--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/uninstalling-cluster-vsphere-installer-provisioned.html

@gnufied @jsafrane @duanwei33 PTAL at the PR and let me whether the NOTE block is placed at the right location.

### CODE REVIEW

**Comment by gnufied** [comment]
lgtm

File: installing/installing_vsphere/uninstalling-cluster-vsphere-installer-provisioned.adoc [hunk 1]
```diff
@@ -8,4 +8,9 @@ toc::[]
 
 You can remove a cluster that you deployed in your VMware vSphere instance by using installer-provisioned infrastructure.
 
+[NOTE]
+====
+Using `openshift-install destroy cluster`, vSphere volumes are not automatically deleted during {product-title} uninstallation. Cluster administrator must manually find vSphere volumes and delete them.
```
Comment (kalexand-rh) [inline installing/installing_vsphere/uninstalling-cluster-vsphere-installer-provisioned.adoc:13]:
```suggestion
When you run the `openshift-install destroy cluster` command to uninstall {product-title}, vSphere volumes are not automatically deleted. The cluster administrator must manually find the vSphere volumes and delete them.
```

**Review by kalexand-rh — COMMENTED** [review]
I have a wording suggestion to make this a little clearer and easier to translate.

File: installing/installing_vsphere/uninstalling-cluster-vsphere-installer-provisioned.adoc (hunk 1 above)
Comment (darshan-nagaraj) [reply installing/installing_vsphere/uninstalling-cluster-vsphere-installer-provisioned.adoc]: @kalexand-rh Fixed. Thanks for the suggestion.

**Pull request merged**